src/frontend/next/src/components/ScrollAction/ScrollAction.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC, ReactChildren, MouseEvent } from 'react';
+
+type ScrollActionProps = {
```

Comment:
What are the cons and pros of `type` and `interface`?

Reply:
@PedroFonsecaDEV : in term of what we need here, they are pretty much doing the same thing. Interface more like about the structure of an object while type is more like an alias of a specific type. Interface can be extended with new class while type cannot. In that sense I would say type is more limited than interface. However, @humphd mentioned that in the next docs they recommend using type over interface so maybe he can answer you better here. 
More about it here: https://blog.logrocket.com/types-vs-interfaces-in-typescript/

Reply:
Interfaces can be extended, and are good for exporting public types.  For internal code, where things won't change once defined, a type is good because it gives a name to a shape/type.  See https://github.com/typescript-cheatsheets/react#types-or-interfaces